### DISCUSSION TITLE
Special Features Grouped by type

### DISCUSSION THREAD

**Discussion by SenorSmartyPants** [jellyfin/jellyfin-web#5757] [Ideas]
As I am adding more special features (19 for one TV season) the list is getting a bit long. **What about adding an option to display special features grouped by extra type?** So instead of just one header of "Special Features" with many items below it, several headers displaying just extras of that type

**Interviews**
Interview 1, Interview 2

**Behind the Scenes**
BTS 1, BTS 2

**Featurettes**
Featurette 1, etc

**Comment by thornbill** [comment]
Here is a related feature request: https://features.jellyfin.org/posts/1940/selectable-sub-menu-for-special-features

In general I would be opposed to adding any new complexity to the legacy views while we are in the midst of migrating tech stack and a pending redesign. (Especially for the item details screen which is already one of the most complex in the app.)

**Comment by TheMelmacian** [comment]
I like the idea.
And it would be nice to have metadata support for movie and season extras.

If it is not feasible to implement it for the legacy view, it should be considered for the new item details component once the current functionality is ported to the new tech stack.

**Reply by SenorSmartyPants** [reply]
You can use nfos for extra metadata now. 

https://github.com/jellyfin/jellyfin/pull/9403
